pkg/platform/api/buildplanner/model/buildexpression.go

```diff
@@ -97,122 +170,255 @@ type BuildExpression struct {
 //	  }
 //	}
 func NewBuildExpression(data []byte) (*BuildExpression, error) {
-	expression := make(map[string]interface{})
-	err := json.Unmarshal(data, &expression)
+	m := make(map[string]interface{})
```

Comment:
Please use words not letters. Letters are fine for very short pieces of code where the intend of the variable is very evident, I don't think this qualifies.